src/dlgIRC.cpp

```diff
@@ -61,265 +90,455 @@ dlgIRC::dlgIRC()
         file.close();
     }
 
-    connection->setNickName(nick);
-    mNick = nick;
-    connection->setUserName("mudlet");
-    connection->setRealName(mudlet::self()->version);
-    connection->setHost("irc.freenode.net");
-    connection->setPort(6667);
+    mNickName = nick;
+    mUserName = "mudlet";
+    mRealName = mudlet::self()->version;
+    mHostName = "irc.freenode.net";
+    mHostPort = 6667;
+    mChannels << "#mudlet";
+
+    connection->setNickName(mNickName);
+    connection->setUserName(mUserName);
+    connection->setRealName(mRealName);
+    connection->setHost(mHostName);
+    connection->setPort(mHostPort);
+
+    connection->sendCommand(IrcCommand::createJoin(mChannels));
     connection->open();
+
+    setupBuffers();
+
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("$ Statring Mudlet IRC Client...")));
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("$ Host: %1:%2").arg(mHostName, QString::number(mHostPort))));
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("$ Nick: %1").arg(mNickName)));
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("$ Auto-Join Channels: %1").arg(mChannels.join(" "))));
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("$ This client supports Auto-Completion using the Tab key.")));
```

Comment:
Yea the config stuff isn't really needed.  I was thinking of adding the other information to the `/help` command instead of where it's at now.